packages/mds-db/events.ts

```diff
@@ -304,6 +304,92 @@ export async function readEventsWithTelemetry({
   }))
 }
 
+// TODO: remove
+// heinous copypasta specifically to provide the VIN with every event, used ONLY by Native.
+// this is to be excised when we dump Native in the garbage in the 1.0 timeframe.
+export async function readEventsWithTelemetryAndVin({
```

Comment:
As much as I prefer the shorter name, we probably wanna go with something like `readEventsWithTelemetryAndVehicleId`. 'vin' isn't _technically_ a thing, though we treat vehicle_id as one.

Reply:
done